matter/si91x/siwx917/BRD4338A/autogen/sl_event_handler.c

```diff
@@ -6,6 +6,8 @@
 #include "rsi_nvic_priorities_config.h"
 #include "rsi_wisemcu_hardware_setup.h"
 #include "sl_device_init_nvic.h"
+#include "sl_simple_button_instances.h"
+#include "sl_simple_led_instances.h"

```

Comment:
I believe they don't have those 2

Reply:
This can get merged, will rebase and correct it in my PR

Reply:
Or @rcasallas-silabs you can remove the BRD4338A changes so that it will directly come from my PR

Reply:
Removed. These changes were merged with [PR#150](https://github.com/SiliconLabs/sdk_support/pull/150).